### DISCUSSION TITLE
Anonymized DNS vs. ODOH?

### DISCUSSION THREAD

**Discussion by tBi2rcAlKQlG** [DNSCrypt/dnscrypt-proxy#2617] [Q&A] [answered]
I've tried out both Oblivious DNS Over HTTPS (ODOH) https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Oblivious-DoH and Anonymized DNS (https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Anonymized-DNS). 

Both seem to work well, and the only functional difference I've observed is that there seem to be more relays for Anonymized DNS vs. ODOH.

I'm curious if there is a move to converge on one of these standards vs. the other?

**Comment by anoduck** [comment, marked as answer]
ODOH uses SSL and was written to natively communicate over TCP. AnonDNS does not use SSL and was written to natively communicate over UDP. 

Either way you split it, ODOH has about 4 consistently reachable servers available on the western hemisphere, two on the eastern. AnonDNS has hundreds.
